infra/aws/terraform/prow-build-cluster/locals.tf

```diff
@@ -81,6 +82,27 @@ locals {
     }
   ]
 
+  karpenter_roles = [
```

Comment:
should we use cluster access entry instead of adding to the aws-auth configmap?

Reply:
That's a good question. I tried this code on the Canary cluster first and could not run Karpenter with the access entry. I may have missed something, so I turned it off and continued with the aws-auth file.